src/OVAL/oval_generator.c

```diff
@@ -138,9 +156,30 @@ void oval_generator_set_timestamp(struct oval_generator *generator, const char *
 	generator->timestamp = oscap_strdup(timestamp);
 }
 
+void oval_generator_update_timestamp(struct oval_generator *generator)
+{
+	time_t et;
+	struct tm *lt;
+	char timestamp[] = "yyyy-mm-ddThh:mm:ss";
```

Comment:
Yeah, you are right. I think I was assuming that it is just a format description.